packages/loopring_v3/contracts/iface/ILoopringV3.sol

```diff
@@ -230,16 +230,30 @@ contract ILoopringV3
         )
         external;
 
-    /// @dev Withdraw
+    /// @dev Requests the withdrawal of the protocol fees from an exchange
     /// @param exchangeId The id of the exchange to withdraw the fees from
-    /// @param tokenAddress The token to withdraw the fees for
-    function withdrawProtocolFees(
+    /// @param tokenAddress The token to withdraw the fees for. Use 0x0 for Ether.
+    function withdrawProtocolFeesFromExchange(
         uint exchangeId,
         address tokenAddress
         )
         external
         payable;
 
+    /// @dev Withdraw all protocol fees to the designated address.
+    ///      Fees first need to be withdrawn from an exchange using
+    ///      withdrawProtocolFeesFromExchange.
+    ///
+    ///      This function can only be called by the owner of this contract.
+    ///
+    /// @param  token The address of the token. Use 0x0 for Ether.
+    /// @param  recipient The address to receive the tokens.
+    function withdrawProtocolFees(
```

Comment:
@Brechtpd , in the future, we'll have a [ProtocolFeeManager](https://github.com/Loopring/protocols/blob/master/packages/loopring_v3/contracts/iface/IProtocolFeeManager.sol), can we actually have a method `setProtocolFeeManager(address pfm)` and then the owner can only withdraw to that address? If the address is 0x0 (default), `withdrawProtocolFees` will be disabled.